addons/binding/org.openhab.binding.knx/META-INF/MANIFEST.MF

```diff
@@ -11,6 +11,8 @@ Import-Package: gnu.io;resolution:=optional,
  org.eclipse.smarthome.config.discovery,
  org.eclipse.smarthome.core.common,
  org.eclipse.smarthome.core.i18n,
+ org.eclipse.smarthome.core.items,
```

Comment:
I think we had this discussion a couple of times already: Bindings must be agnostic of items. Otherwise the whole functional abstraction won't work. So I hope you'll understand this won't pass without a single word of explanation?

Reply:
See https://github.com/openhab/openhab2-addons/pull/3500#discussion_r189028969

I do not use Items directly, but I need their class definitions for the TypeMapper.